File: member-service/src/main/java/com/hedvig/memberservice/commands/UpdateSSNCommand.java
package com.hedvig.memberservice.commands;

import com.hedvig.memberservice.web.dto.Nationality;
import lombok.Value;
import org.axonframework.commandhandling.TargetAggregateIdentifier;

@Value
public class UpdateSSNCommand {
  @TargetAggregateIdentifier long memberId;
    private String SSN;
    private Nationality nationality;
}

File: member-service/src/main/java/com/hedvig/memberservice/commands/UpdateSSNCommand.kt
package com.hedvig.memberservice.commands

import com.hedvig.memberservice.web.dto.Nationality
import org.axonframework.commandhandling.TargetAggregateIdentifier

class UpdateSSNCommand(
    @TargetAggregateIdentifier
    val memberId: Long,
    val ssn: String,
    val nationality: Nationality
)
